terraform/shared/video-player/js/common.js

```diff
@@ -39,15 +40,15 @@ if (vod_ipfs) {
 }
 
 // Configure video player
-var live = videojs('live');
+var live = videojs('live', { liveui: true });
 
 // For any browser except Safari
-if (/^((?!chrome|android).)*safari/i.test(navigator.userAgent) === false) {
-  // Override native player for platform and browser consistency
-  videojs.options.html5.nativeAudioTracks = false;
-  videojs.options.html5.nativeVideoTracks = false;
-  videojs.options.hls.overrideNative = true;
-}
+//if (/^((?!chrome|android).)*safari/i.test(navigator.userAgent) === false) {

```

Comment:
Do we want to delete these altogether? It will be in git history if we ever need it again.